docs/install.rst

```diff
@@ -18,39 +18,71 @@ Requirements
 PlasmaPy requires Python version 3.7 or newer.
 PlasmaPy requires the following packages for installation:
 
-- `NumPy <http://www.numpy.org/>`_ 1.18.1 or newer
-- `SciPy <https://www.scipy.org/>`_ 1.2 or newer
-- `Astropy <http://www.astropy.org/>`_ 4.0 or newer
+- `NumPy <https://www.numpy.org/>`_ — 1.18.1 or newer
+- `SciPy <https://www.scipy.org/>`_ — 1.2 or newer
+- `Astropy <https://www.astropy.org/>`_ — 4.0 or newer
+- `pandas <https://pandas.pydata.org/>`_ — 1.0 or newer
+- `xarray <http://xarray.pydata.org>`_ — above 0.14 but less than 0.16
+- `tqdm <https://tqdm.github.io/>`_ — 4.56 or newer
+- `colorama <https://pypi.org/project/colorama/>`_ — 0.3 or newer
```

Comment:
Why is this still here and in our `setup.cfg` when it was suppose to be dropped in PR #920?